.github/workflows/contract.yml

```diff
@@ -74,15 +63,15 @@ jobs:
           if-no-files-found: error
 
       - name: Download Leap - dev binary
-        uses: AntelopeIO/asset-artifact-download-action@v2
+        uses: AntelopeIO/asset-artifact-download-action@v3
         with:
           owner: AntelopeIO
           repo: leap
-          target: 'v4.0.3'
+          target: 'release/5.0'
```

Comment:
Unfortunately we can't use 5.0.0 because we need AntelopeIO/leap#2055 which will be included in 5.0.1. Clearly this should be changed to 5.0.1+ once 5.0.1 is released.